src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +512,51 @@ g.test('copy_within_same_texture')
     );
   });
 
+// TODO(jiawei.shao@intel.com): test 'stencil-only' is also valid for the texture format 'stencil8'
```

Comment:
This would be easy to add to the test now, we just wouldn't be able to pass it, right? If so we should just add it.

Reply:
```stencil8``` is not defined in gpuweb/types. I've just written the PR https://github.com/gpuweb/types/pull/39 to add it.

Reply:
Done and published a new version of the package, 0.0.34. Feel free to update the test in this PR or a followup.